agent/src/containeroptions/dynamiccontaineroptions.cpp

```diff
@@ -40,4 +40,24 @@ bool DynamicContainerOptions::enableWriteBuffer() const
     return m_enableWriteBuffer;
 }
 
+void DynamicContainerOptions::setEnableTemporaryFileSystemWriteBuffers(bool enabled)
+{
+    m_enableTemporaryFileSystemWriteBuffers = enabled;
+}
+
+bool DynamicContainerOptions::enableTemporaryFileSystemWriteBuffers() const
```

Comment:
I don't think there are any guidelines. I'm also guessing we are mixing a bit, but the Qt style I believe is to have setX(), and just x() to get a property. A true/false check I do think is common to have isX() style though. I'd propose that @frznlogic simply do as how it looks in the code around here, and if there are no good references, check the rest of the code base.